casual/casual-caller/src/main/java/se/laz/casual/connection/caller/ConnectionFactoryLookup.java

```diff
@@ -30,7 +31,7 @@
      * @param qinfo
      * @return a List of 0-n CacheEntries
```

Comment:
This is a single optional now, was a list - is that correct? 
If so update the javadocs to reflect.

Reply:
Yes, this is correct. I'll update the javadocs. Normally there should never be more than one configured source for a queue in a system, but if there is we will always sticky the first one we encounter and continue to use that one (until the server running casual-caller is restarted/the queue cache is cleared). That is apparently how casual itself handles multiple queues of the same name more or less